h2/connection.py

```diff
@@ -684,9 +695,13 @@ def get_next_available_stream_id(self):
         # No streams have been opened yet, so return the lowest allowed stream
         # ID.
         if not self.highest_outbound_stream_id:
+            self.config.logger.debug("Next available stream id: %d",
+                                     1 if self.config.client_side else 2)
             return 1 if self.config.client_side else 2
```

Comment:
Rather than compute this twice, should we put the result in a variable? i.e.

```python
next_stream_id = 1 if self.config.client_side else 2
self.config.logger.debug("Next available stream id: %d", next_available_stream_id)
return next_stream_id
```